src/modules/uv-treeviewleftpanel-module/GalleryView.ts

```diff
@@ -205,9 +205,6 @@ class GalleryView extends BaseView {
         var height = $thumb.data('height');
 
         var $img = $thumb.find('img');
-
-        $img.width(width * this.range);
```

Comment:
These lines were causing image distortion for me when I expanded the left panel -- the thumbnail was stretched to fit the tiles, regardless of its aspect ratio. When I removed the line, I got the expected results (thanks to the max-width/max-height on the img element). I tested in Firefox and IE, but perhaps other browsers have different quirks.